giraffe/src/utils/annotationData.ts

```diff
@@ -35,6 +36,34 @@ const isWithinHoverableArea = (
   )
 }
 
+const distanceToMousePointer = (
+  annotationMark: AnnotationMark,
+  hoverPosition: number
+) => {
+  // check if the Annotation is a range type
+  if (annotationMark.startValue !== annotationMark.stopValue) {
+    if (
+      annotationMark.startValue <= hoverPosition &&
+      hoverPosition <= annotationMark.stopValue
+    ) {
+      // hover position is in the range, effective distance is 0
+      return 0
+    } else {
```

Comment:
nitpick: you don't need the `else`. The whole block can be outside because the above `if` has a return statement

Reply:
Fixed 👍